### DISCUSSION TITLE
Development build in PowerShell?

### DISCUSSION THREAD

**Discussion by bethatasitmay** [PaloAltoNetworks/pan-os-php#671] [Q&A] [answered]
Hi

I know PS isn't supported, but in case you have a suggestion....

To install PAN-OS-PHP in PowerShell, after installing git and php (chocolatey package if it matters), and updating php.ini (change enable_dl off to on), we run the following command from the desired parent directory (c:\tools\pan in my case which installs it to .\pan-os-php):

git clone https://github.com/PaloAltoNetworks/pan-os-php.git

Do you think there might be a way to install the development build this way?  If so, it would allow me to test features in that build.

Thanks!

**Comment by swaschkut** [comment, marked as answer]
development is available here:

git clone https://github.com/swaschkut/pan-os-php.git

**Comment by bethatasitmay** [comment]
Great, thanks!
